float32 acc_x
float32 acc_y
float32 acc_z
float32 gx
float32 gy
float32 gz
float32 gps_long
float32 gps_lat
float32 gps_PDOP
float32 gps_altitute
float32 vel_Left
float32 vel_Right
float32 IMU_TimeStamp
float32 GPS_TimeStamp
